config/initializers/apartment.rb

```diff
@@ -11,8 +11,7 @@
 
   # Add any models that you do not want to be multi-tenanted, but remain in the global (public) namespace.
   # A typical example would be a Customer or Tenant model that stores each Tenant's information.
-  #
-  config.excluded_models = %w{ Account Endpoint SolrEndpoint }
+  config.excluded_models = %w{ Account Endpoint SolrEndpoint RedisEndpoint }
```

Comment:
I'm suspicious that `FcrepoEndpoint` isn't in this list. It's very possible, with STI, we don't need to list out all the subclasses. On the other hand, maybe it's a bug we just haven't noticed yet.